README.md

```diff
@@ -52,84 +53,83 @@ all the config values we need to run this Flex plugin:
 
 Make sure you have [Node.js](https://nodejs.org) as well as [`npm`](https://npmjs.com) installed.
 
-Navigate to the primary plugin folder and run NPM install for the plugin
-```bash
-cd plugin-supervisor-barge-coach
-npm install
-```
+1. Clone this repo:
 
-Navigate to the serverless folder, create and modify the .env file
-```bash
-cd ..
-cd serverless
-***rename the .env.example file to .env and change the below:
-ACCOUNT_SID= Found at https://www.twilio.com/console
-AUTH_TOKEN= Found at https://www.twilio.com/console 
-TWILIO_WORKSPACE_SID = WSXXXXXXXXXXXXXXXXXX
-```
+  ```bash
+  https://github.com/twilio-professional-services/plugin-supervisor-barge-coach
+  ```
 
-Run NPM install for the serverless functions
-```bash
-Run: (from the serverless directory)
-npm install
-```
+2. Install the dependencies
 
-Install the twilio plugin-serverless
-```bash
-Run: 
-twilio plugins:install @twilio-labs/plugin-serverless
-```
+  ```bash
+  # Install the dependencies of the Flex Plugin
+  npm install
+  
+  # Install the dependencies of the Twilio Functions
+  cd functions
+  npm install
+  ```
 
-Deploy the serverless functions into Twilio
-*Do this if you haven't deployed the serverless functions already*
-```bash
-Run: 
-twilio serverless:deploy
-```
-Copy the domain as you'll need this for the .env in the next step
+3. Copy the `.env.example` file in the `functions/` directory:
 
+  ```bash
+  cp functions/.env.example functions/.env
+  ```
 
-From the root plugin directory rename the .env.example file to .env and change the below:
-```bash
-cd ..
-cd plugin-supervisor-barge-coach
+4. Edit `.env`:
 
-var REACT_APP_SERVICE_BASE_URL = 
-Points to the Twilio Function Service URL (example: https://barge-coach-XXXX-dev.twil.io)
+  ```bash
+  ACCOUNT_SID= Found at https://www.twilio.com/console
+  AUTH_TOKEN= Found at https://www.twilio.com/console 
+  TWILIO_WORKSPACE_SID = WSXXXXXXXXXXXXXXXXXX
+  ```
 
-Can be found by by going to https://www.twilio.com/console/functions/overview/services then click on barge-coach (should look like barge-coach-XXXX-dev.twil.io)
+5. Copy the `.env.example` file in the root directory:
 
-var REACT_APP_TASK-CHANNEL_SID =
-Points to Voice Channel SID - Can be found by going to https://www.twilio.com/console/taskrouter/dashboard > click on Workspaces > then Task Channels
-```
+  ```bash
+  cp .env.example .env
+  ```
 
-From the public folder, create the appConfig.js and change on variable within it
-```bash
-cd public
-rename appConfig.example.js to appConfig.js
+6. In a future step, once you have deployed your code, you'll get a domain for your Twilio Functions. Copy this and set the `REACT_APP_SERVICE_BASE_URL` in the `.env` file.
 
-change serviceBaseUrl: "https://barge-coach-XXXX.twil.io"
-```
+7. Copy the `public/appConfig.example.js` over:
+
+  ```bash
+  copy public/appConfig.example.js public/appConfig.js
```

Comment:
```suggestion
  cp public/appConfig.example.js public/appConfig.js
```

Just to be consistent, but what is the actual difference? Follow-up reading: http://ps-2.kev009.com/tl/techlib/manuals/adoclib/cmds/aixcmds1/cp.htm 😛